frontend/lib/js/form-components/SearchBox.js

```diff
@@ -2,38 +2,50 @@ import React                           from 'react';
 import PropTypes                       from 'prop-types';
 import T                               from 'i18n-react';
 import { Creatable as Select }         from 'react-select';
+import { isEmpty } from '../common/helpers';
+import ClearableInput from './ClearableInput';
 
 const SearchBox = (props) => {
-  return (
-    <div className="search-box">
-      <Select
-        name="input"
-        value={props.search.map(t => ({ label: t, value: t }))}
-        options={
-          props.options
-          ? props.options.map(t => ({ label: t, value: t}))
-          : []
-        }
-        onChange={(values) => props.onSearch(values.map(v => v.value))}
-        multi
-        promptTextCreator={(label) => T.translate(
-          'reactSelect.searchFor',
-          { label }
-        )}
-        placeholder={T.translate('reactSelect.searchPlaceholder')}
-        backspaceToRemoveMessage={T.translate('reactSelect.backspaceToRemove')}
-        clearAllText={T.translate('reactSelect.clearAll')}
-        clearValueText={T.translate('reactSelect.clearValue')}
-        noResultsText={T.translate('reactSelect.noResults')}
-      />
-    </div>
-  );
+  return isEmpty(props.mode)
+    ? <div className="search-box">
+        <Select
+          name="input"
+          value={props.search.map(t => ({ label: t, value: t }))}
+          options={
+            props.options
+            ? props.options.map(t => ({ label: t, value: t}))
+            : []
+          }
+          onChange={(values) => props.onSearch(values.map(v => v.value))}
+          multi
+          promptTextCreator={(label) => T.translate(
+            'reactSelect.searchFor',
+            { label }
+          )}
+          placeholder={T.translate('reactSelect.searchPlaceholder')}
+          backspaceToRemoveMessage={T.translate('reactSelect.backspaceToRemove')}
+          clearAllText={T.translate('reactSelect.clearAll')}
+          clearValueText={T.translate('reactSelect.clearValue')}
+          noResultsText={T.translate('reactSelect.noResults')}
+        />
+      </div>
+    : props.mode === 'simple' &&
```

Comment:
You could've made mode a boolean property, that allows for a nice React syntax, i.e. `<SearchBox isMulti />`